cognite/extractorutils/configtools.py

```diff
@@ -503,9 +503,20 @@ def load_yaml(
         InvalidConfigError: If any config field is given as an invalid type, is missing or is unknown
     """
 
-    def env_constructor(_, node):
-        # Expnadvars uses same syntax as our env var substitution
-        return os.path.expandvars(node.value)
+    def env_constructor(_: yaml.SafeLoader, node):
+        bool_values = {
+            "yes": True,
+            "no": False,
+            "true": True,
+            "false": False,
+            "on": True,
+            "off": False,
```

Comment:
While I don't see anyone requiring the string `"true"`, I'm not sure always parsing e.g. `"on"` as `True` is safe

I think sticking with true/false is best

Reply:
hehe took the list from pyyaml bool constructor, I can narrow it down 👍